packages/mui-base/package.json

```diff
@@ -27,31 +27,32 @@
     "url": "https://opencollective.com/mui-org"
   },
   "scripts": {
-    "build": "yarn build:legacy && yarn build:modern && yarn build:node && yarn build:stable && yarn build:types && yarn build:copy-files",
+    "build": "pnpm build:legacy && pnpm build:modern && pnpm build:node && pnpm build:stable && pnpm build:types && pnpm build:copy-files",
     "build:legacy": "node ../../scripts/build.mjs legacy",
     "build:modern": "node ../../scripts/build.mjs modern",
     "build:node": "node ../../scripts/build.mjs node",
     "build:stable": "node ../../scripts/build.mjs stable",
     "build:copy-files": "node ../../scripts/copyFiles.mjs",
     "build:types": "node ../../scripts/buildTypes.mjs",
     "prebuild": "rimraf build tsconfig.build.tsbuildinfo",
-    "release": "yarn build && npm publish build",
+    "release": "pnpm build && pnpm publish",
     "test": "cd ../../ && cross-env NODE_ENV=test mocha 'packages/mui-base/**/*.test.{js,ts,tsx}'",
     "typescript": "tsc -p tsconfig.json",
     "typescript:module-augmentation": "node scripts/testModuleAugmentation.js"
   },
   "dependencies": {
     "@babel/runtime": "^7.23.6",
     "@floating-ui/react-dom": "^2.0.4",
-    "@mui/types": "^7.2.11",
-    "@mui/utils": "^5.15.2",
+    "@mui/types": "workspace:^",
+    "@mui/utils": "workspace:^",
     "@popperjs/core": "^2.11.8",
     "clsx": "^2.0.0",
     "prop-types": "^15.8.1"
   },
   "devDependencies": {
-    "@mui-internal/babel-macros": "^1.0.0",
-    "@mui-internal/test-utils": "^1.0.0",
+    "@mui-internal/babel-macros": "workspace:^",
+    "@mui-internal/test-utils": "workspace:^",
+    "@mui/base": "workspace:*",
```

Comment:
Hmm, do we need to define the same package as its own dependency?

Reply:
Yes, otherwise, we won't be able to access the `@mui/base` contents in tests.

Reply:
Got it!